redaxo/src/core/assets/jquery-ui.custom.min.js

```diff
@@ -1,8 +1,8 @@
-/*! jQuery UI - v1.11.4 - 2015-11-19
+/*! jQuery UI - v1.12.1 - 2020-01-03
 * http://jqueryui.com
-* Includes: core.js, widget.js, mouse.js, position.js, draggable.js, droppable.js, resizable.js, selectable.js, sortable.js
+* Includes: widget.js, position.js, data.js, disable-selection.js, focusable.js, form-reset-mixin.js, jquery-1-7.js, keycode.js, labels.js, scroll-parent.js, tabbable.js, unique-id.js, widgets/draggable.js, widgets/droppable.js, widgets/resizable.js, widgets/selectable.js, widgets/sortable.js, widgets/mouse.js
```

Comment:
Ist das sinngemäß in etwa das gleiche Set, oder haben wir jetzt mehr drin?

Reply:
Wer erinnert sich, wo wir jquery ui überhaupt verwenden?
Oder war das vielleicht auch nur für irgendwelche verworfenen Anfänge, wie der Strukturbaum?

Reply:
Ggf ist das auch nur ein rest der zum dead-code gehört

Reply:
Wobei wir das dann vermutlich besser drin lassen sollten, falls es von Addons verwendet wird.

Reply:
Ok

Reply:
BlÖcks nutzt die Sortables aus jQuery UI: https://github.com/FriendsOfREDAXO/bloecks/blob/c9b34fa63b4f7dd2c91b926a58206ad9cda05c74/plugins/dragndrop/assets_src/js/be/scripts.js#L47

Reply:
Könnten wir bei BlÖcks aber mal umbauen auf ein anderes Sortable, so wie es etwa auch mblock oder yform_usability nutzen.
Aktuell würde ich deshalb jQuery UI aber auch vorerst lieber drin lassen, wie Gregor sagt.

Reply:
Ich habe versucht es sinngemäß zu übernehmen. Wie in der Textdatei angegeben: Core, Interactions + mouse.js ... Effekte habe ich ausgelassen. Mir ist jedoch nur dieser tote Code aufgefallen.

Reply:
Hmm in diesem PR scheinen viel mehr zeilen dazuzukommen als wegzufallen.. sind hier jetzt ggf. Zu viel ui module drinn?

Reply:
(Oder verwirrt hier die anzeige im PR nur und die Dateigrößen sind ungefähr gleich geblieben?)

Reply:
> Hmm in diesem PR scheinen viel mehr zeilen dazuzukommen als wegzufallen..

Der große Zeilenunterschied ist in der jquery.js (die unminifizierte haben wir ja jetzt doch wieder mit drin).
Denke dass das einfach wohl tatsächlich so ist, dass jquery zeilenmäßig so gewachsen ist?

Reply:
Ihr entscheidet. Ich nehm die unminifizierte auch raus. - Wie ihr möchtet.